ROADMAP.md

```diff
@@ -1,74 +1,42 @@
+The Interactive API was originally designed to be a high-level API for OpenFL, but for the cases when more control is required by users, access to lower level interfaces is necessary.
+In OpenFL 1.5, we introduced the Workflow API as an experimental feature, which can be used to specify the federated learning flow, independently of the underlying computing infrastructure. The Workflow API facilitates a seamless transition from local simulation to a federated setting. Additionally, this approach offers greater control over the sequence and content of the FL experiment steps, which enables more complex experiments beyond just horizontal FL. Workflow API also provides more granular privacy controls, allowing the model owner to explicitly permit or forbid the transfer of specific attributes over the network.
 
-The [Task Assigner](https://github.com/securefederatedai/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/assigner.yaml#L7-L9) determines the list of collaborator tasks to be performed, 
-and both in the task runner API as well as the interactive API these tasks can be modified (to varying degrees).
-For example, to perform federated evaluation of a model, only the `aggregated_model_validation` task would be listed for the assigner's block of the federated plan.
-Equivalently for the interactive API, this can be done by only registering a single validation task.
-But there are *many* other types of workflows that can't be easily represented purely by training / validation tasks performed on a collaborator with a single model.
-An example is training a Federated Generative Adversarial Network (GAN); because this may be represented by separate generative and discriminator models, and could leak information about a collaborator dataset,
-the interface we provide should allow for better control over what gets sent over the network and how. 
-Another common request we get is for validation with an aggregator's dataset after training. Today there is not a great way to enable this in OpenFL. 
+### 1.2 Consolidating interfaces
+Today we support four interfaces: TaskRunner API, Workflow API, native Python API, and Interactive API. These are all distinct APIs, and are not particularly interoperable. The strategic vision is to consolidate OpenFL around the Workflow API due to its focus on the needs of the data scientist as the main user of the framework. Over the upcoming 1.x releases, we plan to gradually deprecate and eliminate the legacy APIs, including Python Native API, Interactive API, and ultimately the Task Runner API. OpenFL 2.0 will offer Workflow API exclusively, backed by various runtimes that enable transitioning from a local simulation to a distributed FL experiment, and even setting up a permanent federation (currently only possible via Interactive API).
```

Comment:
* Instead of `Today we support four interfaces...`, I suggest `OpenFL has long since supported multiple ways of running FL experiments, many of which were not interoperable.`
* I suggest not mentioning `...ultimately the TaskRunner API`, unless we have migrated away from all TR APIs ourselves. There are too many hooks to that interface (including workflow API itself). It is ok to call out only interactive and native API deprecations explicitly, IMO.